app/ts/library/flashbots.ts

```diff
@@ -180,18 +179,21 @@ export async function sendBundle(bundle: Bundle, targetBlock: bigint, fundingAmo
 				id: index,
 				params: [transaction]
 			}))
-			if (network.mempoolSubmitRpcEndpoint === undefined) throw new Error('MemPool Submit Rpc Endpoint is not set')
-			const requests = await Promise.all(payloads.map(async (payload) => {
-				if (network.mempoolSubmitRpcEndpoint === undefined) throw new Error('MemPool Submit Rpc Endpoint is not set')
-				return await fetch(network.mempoolSubmitRpcEndpoint, { method: 'POST', body: payload, headers: { 'Content-Type': 'application/json' } })
-			}))
-			for (const request of requests) {
-				const response = await request.json()
-				if (response.error !== undefined && response.error !== null) {
-					throw new Error(response.error.message)
+			if (network.mempoolSubmitRpcEndpoint === undefined) throw new Error('MemPool Submit RPC Endpoint is not set')
+			for (const payload of payloads) {
+				const MAX_ATTEMPTS = 40
```

Comment:
This is only 10 seconds at 250ms per attempt.  If the blockchain has slow block times or it is backlogged/slow, halting may cause even more problems.  I'm not sure what the right answer here is, but maybe at least wait for 30 seconds since I don't think any EVM like chains have block times longer than 30 seconds?

We could also have a backoff after 40 tries, or a cooldown.

Reply:
I'll change the timeout to setTimeout(r, 50 + attempt* 50), so its faster at start and gets gradually slower, this is around 40 secs